common/static/common/js/spec_helpers/discussion_spec_helper.js

```diff
@@ -65,44 +73,50 @@
\'forum-nav-browse-menu-item\'>\n                <a href=\'#\' class=\'forum-nav-browse-title\'>Parent</a>\n                <ul class=\'forum-nav-browse-submenu\'>\n                    <li class=\'forum-nav-browse-menu-item\'>\n                        <a href=\'#\' class=\'forum-nav-browse-title\'>Target</a>\n                        <ul class=\'forum-nav-browse-submenu\'>\n                            <li\n                                class=\'forum-nav-browse-menu-item\'\n                                data-discussion-id=\'child\'\n                                data-cohorted=\'false\'\n                            >\n                                <a href=\'#\' class=\'forum-nav-browse-title\'>Child</a>\n                            </li>\n                        </ul>\n                    <li\n                        class=\'forum-nav-browse-menu-item\'\n                        data-discussion-id=\'sibling\'\n                        data-cohorted=\'false\'\n                    >\n                        <a href=\'#\' class=\'forum-nav-browse-title\'>Sibling</a>\n                    </li>\n                </ul>\n            </li>\n            <li\n                class=\'forum-nav-browse-menu-item\'\n                data-discussion-id=\'other\'\n                data-cohorted=\'true\'\n            >\n                <a href=\'#\' class=\'forum-nav-browse-title\'>Other Category</a>\n            </li>\n        </ul>\n    </div>\n    <div class=\'forum-nav-thread-list-wrapper\' id=\'sort-filter-wrapper\' tabindex=\'-1\'>\n        <div class=\'forum-nav-refine-bar\'>\n            <label class=\'forum-nav-filter-main\'>\n                <select class=\'forum-nav-filter-main-control\'>\n                    <option value=\'all\'>Show all</option>\n                    <option value=\'unread\'>Unread</option>\n                    <option value=\'unanswered\'>Unanswered</option>\n                    <option value=\'flagged\'>Flagged</option>\n                </select>\n            </label>\n            <% if (isCohorted && isPrivilegedUser) { %>\n            <label class=\'forum-nav-filter-cohort\'>\n                <span class=\'sr\'>Cohort:</span>\n                <select class=\'forum-nav-filter-cohort-control\'>\n                    <option value=\'\'>in all cohorts</option>\n                    <option value=\'1\'>Cohort1</option>\n                    <option value=\'2\'>Cohort2</option>\n                </select>\n            </label>\n            <% } %>\n            <label class=\'forum-nav-sort\'>\n                <select class=\'forum-nav-sort-control\'>\n                    <option value=\'activity\'>by recent activity</option>\n                    <option value=\'comments\'>by most activity</option>\n                    <option value=\'votes\'>by most votes</option>\n                </select>\n            </label>\n        </div>\n    </div>\n    <div class=\'search-alerts\'></div>\n    <ul class=\'forum-nav-thread-list\'></ul>\n</script>");
```

Comment:
Not anything we can fix in this PR, but holy crap, JS tests should never do this inline.